src/main/java/org/resitrack/controller/HouseController.java

```diff
@@ -1,4 +1,178 @@
 package org.resitrack.controller;
 
+import org.resitrack.entity.House;
+import org.resitrack.util.CommonUtil;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Create a House Controller.
+ *
+ * @author TuDucThinh
+ * @since 13/3/2023
+ */
 public class HouseController {
+    /**
+     * Used to access Scanner.
+     */
+    private final Scanner scanner;
+    /**
+     * Used to access townController.
+     */
+    private TownController townController;
+    /**
+     * Used to create a list of houses.
+     * Helps users to store houses that have just been saved to town
+     */
+    private List<House> houses;
+
+    public HouseController() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void setTownController(TownController townController) {
+        this.townController = townController;
+    }
+
+    public void setHouses(List<House> houses) {
+        this.houses = houses;
+    }
+
+    /**
+     * This function use for create a new house inside a town.
+     * After entering, you will be notified as follows: Successfully add the house
+     * If you want to add another house, press 1 if you want to exit, press 0
+     */
+    public void createNewHouse() {
+
+        String townId = getTownId();
+        String townName = townController.getTownById(townId).getName();
```

Comment:
As we discussed before, we should create a function that returns a valid Town. A Town contains id and name so we don't need to access into town list twice time to get information of same object